backend/src/schema/resolvers/users.js

```diff
@@ -34,4 +103,43 @@ export default {
       return neo4jgraphql(object, params, context, resolveInfo, false)
     },
   },
+  User: {
+    ...undefinedToNull([
+      'actorId',
+      'avatar',
+      'coverImg',
+      'deleted',
+      'disabled',
+      'locationName',
+      'about',
+    ]),
+    ...count({
+      contributionsCount: '-[:WROTE]->(related:Post)',
+      friendsCount: '<-[:FRIENDS]->(related:User)',
+      followingCount: '-[:FOLLOWS]->(related:User)',
+      followedByCount: '<-[:FOLLOWS]-(related:User)',
+      commentsCount: '-[:WROTE]->(r:Comment)',
+      commentedCount: '-[:WROTE]->(:Comment)-[:COMMENTS]->(related:Post)',
+      shoutedCount: '-[:SHOUTED]->(related:Post)',
+      badgesCount: '<-[:REWARDED]-(related:Badge)',
+    }),
```

Comment:
It was really interesting and fun to see how you did that stuff @roschaefer 🤗😎

Reply:
agreed!